### PR TITLE
bump dependency versions

### PR DESCRIPTION
**Pull request by nllong**
Bump locked versions

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -28,32 +28,31 @@ packages = [
...
-pre-commit = "2.7.1"
-pytest = "6.1.0"
-pytest-cov = "2.11.1"
-sphinx = "4.5.0"
-sphinx_rtd_theme = "0.5.0"
-sphinx-jsonschema = "1.16.7"
-syrupy = "^1.5.0"
+autopep8 = "1.6.0"
+coveralls = "3.3.1"
+#flake8 = ">=4.0.1"
```
Comment (vtnate) [inline pyproject.toml:47]: Is this supposed to be commented?

File: pyproject.toml [hunk 2]
```diff
@@ -28,32 +28,31 @@ packages = [
...
+mypy = "^0.961"
+pre-commit = "2.19.0"
+pytest = "7.1.2"
+pytest-cov = "3.0.0"
+sphinx = "5.0.1"
```
Comment (vtnate) [inline pyproject.toml:52]: Has this changed to esbonio?

**Review by vtnate — CHANGES_REQUESTED** [review]
Besides my 2 small questions, LGTM.

File: pyproject.toml (hunk 1 above)
Comment (nllong) [reply pyproject.toml:47]: yeah, i just removed it. We use the flake8 that is in pre-commit now.

File: pyproject.toml (hunk 2 above)
Comment (nllong) [reply pyproject.toml:52]: I don't think so. Esbonio looks like a language server for sphinx? https://pypi.org/project/esbonio/.. says early development too.

**Review by vtnate — APPROVED** [review]

**Pull request merged**